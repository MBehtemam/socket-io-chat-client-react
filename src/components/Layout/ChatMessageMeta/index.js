import styled from "styled-components";

const ChatMessageMeta = styled.div`
  display: flex;
  flex-direction: row;
  justify-content: space-between;
`;
export default ChatMessageMeta;
