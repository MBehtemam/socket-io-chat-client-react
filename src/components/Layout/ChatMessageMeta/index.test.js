import React from "react";
import styled from "styled-components";
import renderer from "react-test-renderer";
import "jest-styled-components";
import ChatMessageMeta from "./index";

describe("Test Suits for Chat Message Meta", () => {
  test("for matching snapshot", () => {
    const tree = renderer.create(<ChatMessageMeta />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
