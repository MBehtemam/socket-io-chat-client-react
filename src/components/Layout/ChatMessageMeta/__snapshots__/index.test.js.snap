// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Suits for Chat Message Meta for matching snapshot 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

<div
  className="c0"
/>
`;
